options/locale/locale_en-US.ini

```
@@ -1005,6 +1005,16 @@ file_view_rendered = View Rendered
 file_view_raw = View Raw
 file_permalink = Permalink
 file_too_large = The file is too large to be shown.
+bidi_bad_header = `This file contains unexpected Bidrectional Unicode characters!`
+bidi_bad_description = `This file contains unexpected Bidrectional Unicode characters that may be processed differently from what appears below.<br>If your use case is intentional and legitimate, you can safely ignore this warning.<br>Use the Escape button to reveal hidden characters.`
```

Typo, I will fix it in my PR together.

```suggestion
bidi_bad_description = `This file contains unexpected Bidirectional Unicode characters that may be processed differently from what appears below.<br>If your use case is intentional and legitimate, you can safely ignore this warning.<br>Use the Escape button to reveal hidden characters.`
```